src/main/java/duke/exception/DukeInvalidSavedToFileLineType.java

```diff
@@ -0,0 +1,12 @@
+package duke.exception;
+
+/**
+ * Handles unknown command entered.
+ */
+public class DukeInvalidSavedToFileLineType extends DukeException {
```

Comment:
```suggestion
public class DukeInvalidSavedToFileLineTypeException extends DukeException {
```
Do you think it would be better if you can state "Exception" in the variable name to show the meaning more explicitly?

Reply:
Yes! Thanks for the suggestion.